### ISSUE TITLE
Slider Posts Carousel always in 2 cols

### ISSUE THREAD

**Issue by eric-ppt** [dudaster/ele-custom-skin#653]
Elemtnor 3.5.3
Customer skin pro  3.1.0
if i set 4 or other number in slider to show . only show 2 in page .
all the sites have the same bugs

https://s3.bmp.ovh/imgs/2022/01/fee328a48923c27a.png

**Comment by mocana** [comment]
I am also having this issue on several websites. The setting to dictate how many posts to display at a given size of screen is unresponsive.

![screen 2022-01-07 at 8 11 21 AM](https://user-images.githubusercontent.com/6290157/148563892-2b6bfe69-bec1-4b39-bd5d-d218781599ed.png)
.

**Comment by elliotmersch** [comment]
Same here, I got the pro version specifically for this

**Comment by mocana** [comment]
Yeah I'm finding on just about every site we've installed. We keep plugins updated weekly so a lot are now "broken" in this respect. Hope we get a hot fix soon.
